lib/auxiliary/github.rb

```diff
@@ -39,6 +39,16 @@ def pr_closed?(pr_object = find_pr)
       pr_object.state == 'closed'
     end
 
+    def pr_check_run(pr_object = find_pr)
+      unless (pr_object['title'] =~ /SKIP-HCK-CI/).nil? && (pr_object['body'] =~ /SKIP-HCK-CI/).nil?
```

Comment:
Maybe use config file with dictionary?

Just asking if we would like in th future to have different behavior related to trigger file. Like don't trigger all tests , only specific driver 

Not a must - just to think about for the future